x/twap/twapmodule/module.go

```diff
@@ -15,6 +16,10 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 
+	"github.com/osmosis-labs/osmosis/v10/x/twap"
+	twapclient "github.com/osmosis-labs/osmosis/v10/x/twap/client"
```

Comment:
nit: why would we want to rename the default 'client` to `twapclient` when there is no clash in names?

Reply:
Theres a client import further up `	"github.com/cosmos/cosmos-sdk/client"`